docs/dev/contributing.md

```diff
@@ -0,0 +1,87 @@
+Contributing guidelines
+=======================
+
+<!--
+    Style rule: one sentence per line please!
+    This makes git diffs easier to read.
+-->
+
+This guide explains how to contribute to the TeleIRC project.
+It explicitly defines working practices of the development team.
+This document helps new contributors get up to speed with working on the project.
+It is a living document and will change.
+If you think something could be better, please [open an issue](https://github.com/RITlug/teleirc/issues/new/choose) with your feedback.
+
+
+## Table of contents
+
+1. [Set up a development environment](#set-up-a-development-environment)
+2. [Open a new pull request](#open-a-new-pull-request)
+3. [Maintainer response time](#maintainer-response-time)
+
+
+## Set up a development environment
+
+**Contents**:
+
+1. [Requirements](#requirements)
+2. [Create Telegram bot](#create-telegram-bot)
+3. [Create Telegram group](#create-telegram-group)
+4. [Configure and run TeleIRC](#configure-and-run-teleirc)
+
+### Requirements
+
+To set up a TeleIRC development environment, you need the following:
+
+* [Nodejs](https://nodejs.org/en/) (v10+ preferred)
+* Telegram account
+* IRC client ([HexChat](https://hexchat.github.io/) recommended)
+* For docs: [Python 3](https://www.python.org/downloads/) (3.6+ preferred)
+
+### Create Telegram bot
+
+Create a Telegram bot using the Telegram [BotFather](https://t.me/botfather).
+See [TeleIRC documentation](/quick-install) for more instructions on how to do this.
```

Comment:
Huh, I'm not familiar with this syntax! I'll assume you tested that the link works :P